packages/@react-spectrum/toast/src/ToastContainer.tsx

```diff
@@ -112,19 +106,39 @@ export function ToastContainer(props: SpectrumToastContainerProps): ReactElement
   let activeToastContainer = useActiveToastContainer();
   let state = useToastQueue(getGlobalToastQueue());
 
+  let {placement, isCentered} = useMemo(() => {
+    let placements = (props.placement ?? 'bottom').split(' ');
+    let placement = placements[placements.length - 1];
+    let isCentered = placements.length === 1;
+    return {placement, isCentered};
+  }, [props.placement]);
+
   if (ref === activeToastContainer && state.visibleToasts.length > 0) {
     return (
       <Toaster state={state} {...props}>
         <ol className={classNames(toastContainerStyles, 'spectrum-ToastContainer-list')}>
-          {state.visibleToasts.slice().reverse().map((toast) => (
-            <li
-              key={toast.key}
-              className={classNames(toastContainerStyles, 'spectrum-ToastContainer-listitem')}>
-              <Toast
-                toast={toast}
-                state={state} />
-            </li>
-          ))}
+          {state.visibleToasts.slice().reverse().map((toast, index) => {
+            let shouldFade = isCentered && index !== 0;
+            return (
+              <li
+                key={toast.key}
+                className={classNames(toastContainerStyles, 'spectrum-ToastContainer-listitem')}
+                style={{
+                  // @ts-expect-error
+                  viewTransitionName: `_${toast.key.slice(2)}`,
```

Comment:
we can probably use the actual key if we apply https://developer.mozilla.org/en-US/docs/Web/API/CSS/escape_static when we generate the random id

Reply:
Tried this initially, and it broke animations. Likely a browser bug.

Reply:
If it ever return `0`, this slice won't work though, I don't think it'd stringify to `0.000`